### PR TITLE
Add spec for registration error due to duplicate email

### PR DESCRIPTION
**Pull request by swrobel**
Just wanted to add a (passing) spec for the situation where an error is raised on registration. Thanks for the great gem!

### CODE REVIEW

**Comment by Gokul595** [comment]
@swrobel Thanks for the pull request 👍

File: spec/dummy/spec/requests/users/registration_requests_spec.rb [hunk 1]
```diff
@@ -31,6 +31,16 @@
         expect(response.headers['Expire-At']).to be_present
         expect(response.headers['Refresh-Token']).to be_present
       end
+
+      it 'should return 422 - email has already been taken' do
```
Comment (Gokul595) [inline spec/dummy/spec/requests/users/registration_requests_spec.rb:35]: Can you move this to invalid params context?

Comment (swrobel) [reply spec/dummy/spec/requests/users/registration_requests_spec.rb:35]: I thought about putting it there, but it's not technically invalid params. The params are valid, it's just a validation error.

Comment (Gokul595) [reply spec/dummy/spec/requests/users/registration_requests_spec.rb:35]: Got it. I have created the invalid params context for grouping all invalid cases. Maybe we need to change the context name, I will merge this now and change the context name later. Thanks.

**Pull request merged**